packages/gas-fee-controller/src/gas-util.ts

```diff
@@ -101,7 +108,9 @@ export async function fetchLegacyGasPriceEstimates(
     mode: 'cors',
     headers: {
       'Content-Type': 'application/json',
-      ...(clientId && makeClientIdHeader(clientId)),
+      Authorization: `Basic ${infuraAuthToken}`,
```

Comment:
Is it worth a `#getHeaders` to avoid this duplication?

This one also has the `Content-Type` but I can't imagine it causes issue on the other request?

We can test of course.

Reply:
Fixed in https://github.com/MetaMask/core/pull/4068/commits/18f7461502703dcc7a2fa185b3a6258dfd28ba7c